nvidia_deeplearningexamples_tacotron2.md

```diff
@@ -0,0 +1,54 @@
+---
+layout: hub_detail
+background-class: hub-background
+body-class: hub
+title: Tacotron 2
+summary: The Tacotron 2 model for generating mel spectrograms from text
+category: researchers
+image: nvidia_logo.png
+author: NVIDIA
+tags: [audio]
+github-link: https://github.com/NVIDIA/DeepLearningExamples/tree/master/PyTorch/SpeechSynthesis/Tacotron2
+featured_image_1: no-image
+featured_image_2: no-image
+---
+
+```python
+import torch
+hub_model = torch.hub.load(github='nvidia/DeepLearningExamples', model='nvidia_tacotron2')
+```
+will load the Tacotron2 model pre-trained on [LJ Speech dataset](https://keithito.com/LJ-Speech-Dataset/)
+
+### Model Description
+
+The Tacotron 2 and WaveGlow model form a text-to-speech system that enables user to synthesise a natural sounding speech from raw transcripts without any additional prosody information. The Tacotron 2 model produces mel spectrograms from input text using encoder-decoder architecture. WaveGlow (also available via torch.hub) is a flow-based model that consumes the mel spectrograms to generate speech.
+
+This implementation of Tacotron 2 model differs from the model described in the paper. Our implementation uses Dropout instead of Zoneout to regularize the LSTM layers.
+
+### Example
+
+Here's a sample execution on a dummy input:
+
+```python
+import torch
+print('\nLoading tacotron2 model from torch.hub.')
+hub_model = torch.hub.load(github='nvidia/DeepLearningExamples', model='nvidia_tacotron2')
+hub_model = hub_model.cuda()
+hub_model.eval()
+inp = torch.randint(low=0, high=148, size=(1,140), dtype=torch.long)
```

Comment:
describe what this input is supposed to be, for example see https://pytorch.org/hub/facebookresearch_pytorch-gan-zoo_dcgan/

Reply:
changed strategy of this example to go from plain text to sound using the two torch.hub models in series